third-party/realdds/doc/DDS ICD.md

```diff
@@ -0,0 +1,147 @@
+# realdds Topics
+
```

Comment:
The intention of this document is not to teach DDS, but to enable users to use our topics.
Some explanations will be added in the readme file mentioned before, more in the use-cases file, but it won't be a complete guide to DDS :-)

Reply:
I understand. The fact is that I did not know that you intended to write other documentation about it.
The whole aim is to see that with the documentation you provide, a user will be able to use the dds wrapper.
Now that you tell me that other docs will be released, it would be great for me that you explain me which docs will be released, and what is their scope.
Otherwise, I feel like I just check the English :-)

Reply:
As discussed, this document will list the topics and their structure. A use-case document will be added soon, describing the actions this library supports and listing the topics sent for each use-case. A user guide, maybe with code examples, will be added in the future.